src/SimpleExcelReader.php

```diff
@@ -314,15 +325,17 @@ protected function getSheet(): SheetInterface
         $this->reader->open($this->path);
 
         foreach ($this->reader->getSheetIterator() as $key => $sheet) {
-            if ($key === $this->sheetNumber) {
+            if (($this->sheetNumber !== -1 && $key === $this->sheetNumber) || ($this->sheetName != "" && $this->sheetName === $sheet->getName())) {
```

Comment:
To my eyes, this condition is unreadable. Could you extract it to a function with a sane name?

Reply:
Here's a video with an example: https://freek.dev/1955-greatly-improve-the-readability-of-your-code-by-breaking-down-complex-conditionals